src/topology/algebra/ordered.lean

```diff
@@ -318,6 +318,158 @@ begin
   exact hs.Icc_subset ys zs ⟨le_of_lt hy, le_of_lt hz⟩
 end
 
+/-!
+### Neighborhoods to the left and to the right on an order_closed_topology
+
+Limits to the left and to the right of real functions are defined in terms of neighborhoods to
+the left and to the right, either open or closed, i.e., members of `nhds_within a (Ioi a)` and
+`nhds_wihin a (Ici a)` on the right, and similarly on the left. Here we simply prove that all
+right-neighborhood of a point are equal, and we'll prove later other useful characterizations which
+require the stronger hypothesis `order_topology α` -/
+
+-- NB: If you extend the list, append to the end please to avoid breaking the API
```

Comment:
Hmm, maybe `tfae` should be changed to allow referencing statements by user-provided names, rather than numbers.

Reply:
Oh maybe. To be honest (I was about to edit the PR description to explain what I did with this), I just copy-pasted a comment that was already in a later tfae